courses/machine_learning/production_ml_systems/0_export_data_from_bq_to_gcs.ipynb

```diff
@@ -0,0 +1,248 @@
+{
```

Comment:
<p>shouldn;'t need this on AI Platform</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/584/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Done!</p>